pandas/core/strings/accessor.py

```diff
@@ -2409,6 +2419,17 @@ def get_dummies(self, sep: str = "|"):
         ----------
         sep : str, default "|"
             String to split on.
+        prefix : str, list of str, or dict of str, default None
+            String to append DataFrame column names.
+            Pass a list with length equal to the number of columns
+            when calling get_dummies on a DataFrame. Alternatively, `prefix`
+            can be a dictionary mapping column names to prefixes.
+        prefix_sep : str, default '_'
+            If appending prefix, separator/delimiter to use.
+        dummy_na : bool, default False
+            Add a column to indicate NaNs, if False NaNs are ignored.
```

Comment:
It seems to me users can already do this in a straightforward manner:

    pd.concat([ser.str.get_dummies(), ser.isna().rename("NaN")], axis=1)

Is this not sufficient?